examples/lfric/eg15/matvec_opt.py

```diff
@@ -89,8 +89,10 @@ def trans(psy):
                 # optimisations will go here.
                 kernel_schedule.view()
                 result = fortran_writer(kernel_schedule)
-                print (result)
+                print(result)
                 # Abort after the first matrix vector kernel for the
-                # time being.
+                # time being.]
+                print ("Aborting to view the modifications to the matrix "
```

Comment:
Erroneous "]" and space before brackets!

Reply:
oops again.

Reply:
That pesky white space is still there...

Reply:
I forgot to change this  - I just read the first part - apologies.